tests/e2e/account/pw_register_test.py

```diff
@@ -0,0 +1,195 @@
+"""Test account registration."""
+
+import pytest
+from playwright.sync_api import Page, expect
+from pytest_bdd import given, parsers, scenarios, then, when
+
+scenarios('account/register.feature')
+pytestmark = pytest.mark.usefixtures('live_server')
+
+TWOFLOWER_EMAIL = 'twoflower@example.org'
+TWOFLOWER_PHONE = '+12015550123'
+TWOFLOWER_PASSWORD = 'te@pwd3289'  # nosec
+
+
+def check_recaptcha_loaded(page: Page) -> None:
```

Comment:
This should be `wait_until_recaptcha_loaded` and it should return a bool indicating success or failure.